pkg/controller/direct/compute/forwardingrule_controller.go

```diff
@@ -358,6 +360,51 @@ func (a *forwardingRuleAdapter) Create(ctx context.Context, createOp *directbase
 		return fmt.Errorf("getting ComputeForwardingRule %q: %w", a.fullyQualifiedName(), err)
 	}
 
+	// Set labels for the created forwarding rule
```

Comment:
Should we update the mock to reflect the reality of what GCP does (i.e. set labels to nil in the Create method?)